src/components/src/test/java/org/apache/jmeter/timers/SyncTimerTest.java

```diff
@@ -88,8 +88,7 @@ public void testTimerWithInvalidTimeout() {
     private long timeDelay(SyncTimer timer) {
         long start = System.currentTimeMillis();
         timer.delay();
-        long duration = System.currentTimeMillis() - start;
-        return duration;
+        return System.currentTimeMillis() - start;
```

Comment:
Technically speaking, those "one-time" variables might help to debug the values.
I can set a breakpoint "at the return statement" and see the delay value.
As the variable is inlined the same breakpoint might become harder.

I don't see how this change simplifies JUnit5->JUnit4 migration :-/

Reply:
Yeah you are correct, this one doesn't relate to JUnit5, was just a simplification of the code, as I don't think the name `duration` is especially helpful given the method is short and called `timeDelay`. Regarding debugging, you could also just step out of the function and again as it's a shot method this one didn't seem like it would need a lot of debugging.

Happy to revert this change it you'd prefer.

Reply:
I see where are you heading, and those small changes are good.

However, they might interfere with other PRs. See https://github.com/apache/jmeter/pull/484

The title says "whitespace, comments tidy", and variable inlining is neither whitespace nor comments.

My opinion here would be:
1) It is ok to keep this change in this PR (it is unlikely someone would face a merge conflict over this particular line)
2) Please refrain from making unrelated changes in your further contributions. Here you have **single** commit that touches lots of lines. Suppose someone would be bitten by a merge conflict. Of course, they might try to diff what "you intended to change", and they might be overwhelmed by 5000 line diff of **different** styles of changes.

In other words, it is OK to have 5000 lines diff that implements "remove trailing semicolon" action (so even in case of merge conflicts it is clear what should be done), however, it is not OK to have 1000 lines diff with three unrelated changes in a single commit.


@pmouawad , comments, thoughts?

Reply:
Yes, this makes sense, thank you for taking the time to explain; I will try to keep changes specific to a single PR. I'll remove this change for now.